### PR TITLE
Sharpen README: logo, badges, feature overview

### PR DESCRIPTION
**Pull request by kirilklein**
- Add the BONSAI logo at the top (`docs/bonsai_logo.png`).
- Badges: JOSS status, CI badges repointed from the old `FGA-DIKU/EHR` name to `FGA-DIKU/BONSAI`, Python and license.
- Short "What it does" overview and the existing pipeline figure.
- Quickstart with the same five commands, one-line comments, plus the `flash_attn` extra.
- Outcome file schema as a table, resume-training and related projects (CORE-BEHRT, PHAIR_EHR).

No code changes.

## Summary by CodeRabbit

* **Documentation**
  * Reorganized the README with a clearer project overview, branding, and status badges.
  * Added quickstart guidance for data creation, pretraining, outcomes, finetuning, and training.
  * Added custom-data instructions, including an outcome table and resume-training example.
  * Clarified supported causal and masked prediction approaches, output locations, and checkpoint paths.
  * Added links to related projects.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,67 +1,93 @@
...
+  <a href="https://github.com/FGA-DIKU/BONSAI/actions/workflows/unittests.yml"><img src="https://github.com/FGA-DIKU/BONSAI/actions/workflows/unittests.yml/badge.svg" alt="Unit tests"></a>
+  <a href="https://github.com/FGA-DIKU/BONSAI/actions/workflows/lint.yml"><img src="https://github.com/FGA-DIKU/BONSAI/actions/workflows/lint.yml/badge.svg" alt="Lint"></a>
+  <a href="https://github.com/FGA-DIKU/BONSAI/actions/workflows/format.yml"><img src="https://github.com/FGA-DIKU/BONSAI/actions/workflows/format.yml/badge.svg" alt="Format"></a>
+  <img src="https://img.shields.io/badge/python-3.12-blue" alt="Python 3.12">
+  <a href="LICENSE"><img src="https://img.shields.io/badge/license-MIT-green" alt="MIT"></a>
+</p>
 
-BONSAI helps researchers and data scientists preprocess EHR data, train models, and generate outcomes for downstream clinical predictions and analyses.
+BONSAI turns raw EHR data into tokenized patient histories, pretrains a BERT-style encoder on them, and finetunes it to predict clinical outcomes. It is the successor of [CORE-BEHRT](https://github.com/mikkelfo/CORE-BEHRT), rebuilt on PyTorch Lightning and Hydra for large-scale, reproducible experiments.
```
Comment (mikkelfo) [inline README.md]: We don't train exclusively train a BERT-style model. Just a transformer (BERT or GPT)

File: README.md [hunk 2]
```diff
@@ -1,67 +1,93 @@
...
 
-### Setup (requires Python 3.12)
+## What it does
 
-```
+- **Data creation.** Reads [MEDS](https://github.com/Medical-Event-Data-Standard/meds)-formatted records and builds temporally ordered sequences of medical concepts with age, time and segment features. Numeric values can be binned into tokens; concepts can be merged or dropped by regex.
+- **Pretraining.** Masked language modeling with a ModernBERT-style encoder (RoPE, pre-norm, flash-attention or PyTorch SDPA backend) implemented in the repo, no HuggingFace dependency.
```
Comment (mikkelfo) [inline README.md]: We don't use ModernBERT anymore

File: README.md [hunk 3]
```diff
@@ -1,67 +1,93 @@
...
-We use the [finetune.yaml](./configs/examples/example_finetune.yaml) config to have a short resource-light training that can run locally and point it to the dataset created in step 1, the checkpoint created in step 2, and the labels created in step 3.
+# 5. Or train from scratch, skipping step 2
+python bonsai/run/train.py --config-name examples/example_finetune dataset=correlated_MEDS_data outcome=examples/example_outcome1
+```
 
-5. Train model.
-`python bonsai/run/train.py --config-name examples/example_finetune dataset=correlated_MEDS_data outcome=examples/example_outcome1`
-We use the [finetune.yaml](./configs/examples/example_finetune.yaml) config to have a short resource-light no-pretraining training that can run locally and point it to the dataset created in step 1 and the labels created in step 3.
+The example configs live in [`configs/examples/`](configs/examples/). Every value can be overridden on the command line with Hydra's `key=value` syntax; the `*_val` variants show the pipeline with numeric lab values.
```
Comment (mikkelfo) [inline README.md:63]: Worth a discussion whether this minimal version is preferred over the previous?

**Review by mikkelfo — COMMENTED** [review]
Sharper than before, but also less explaination. Worth a discussion imo, as it's not given that sharper is better (depending on audience) @Montgomeryyyy @Sllambias

File: README.md (hunk 3 above)
Comment (Sllambias) [reply README.md:63]: I think this is much nicer. I'm of the very strong belief that short means that there's a slight chance that people actually read the docs

File: README.md [hunk 4]
```diff
@@ -1,67 +1,93 @@
...
-`python bonsai/run/pretrain.py --config-name examples/example_pretrain dataset=correlated_MEDS_data paths.ckpt_path=/path/to/run_id_1234/ckpt.last run_id=1234`
+**Resume training** by passing the checkpoint and the original run id. Without `run_id` a new run directory is created.
+
+```bash
+python bonsai/run/pretrain.py --config-name examples/example_pretrain dataset=correlated_MEDS_data paths.ckpt_path=/path/to/run_id_1234/ckpt.last run_id=1234
+```
 
-## Outcomes creation
-We provide a standardized script to generate outcomes in [create_outcome.py](/bonsai/run/create_outcome.py), however you can also provide your own, in case our script doesn't accommodate your needs. 
+The pre-Lightning version is preserved at the `pre-lightning` tag.
```
Comment (Sllambias) [inline README.md]: I think we can remove this from the readme

File: README.md [hunk 5]
```diff
@@ -1,67 +1,93 @@
...
+| column | meaning |
+|---|---|
+| `subject_id` | the person of interest |
+| `split` | `train`, `tuning` or `held_out`; one file covers all splits |
+| `outcome_date` | when the outcome happened, nullable |
+| `index_date` | when prediction starts, required |
+| `censor_date` | data cutoff, required |
 
-`python bonsai/run/pretrain.py --config-name examples/example_pretrain dataset=correlated_MEDS_data paths.ckpt_path=/path/to/run_id_1234/ckpt.last run_id=1234`
+**Resume training** by passing the checkpoint and the original run id. Without `run_id` a new run directory is created.
```
Comment (Sllambias) [inline README.md]: This gets grouped under "bring your own data headline now, should probably have its own or be put somewhere more appropriate

**Review by Sllambias — CHANGES_REQUESTED** [review]
For me just minor changes requested (including Mikkels that we're not exclusively using Bert) and then good to approve. 
I think it's much clearer and also the formatting is less bloated to look at, so I think there's a much higher chance that users will read at least a bit of the readme :)

File: README.md (hunk 3 above)
Comment (Montgomeryyyy) [reply README.md:63]: I also think this looks good and it very intuitive.

**Review by Montgomeryyyy — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (kirilklein) [reply README.md]: Nice, didn't know it was implemented already:)

File: README.md (hunk 2 above)
Comment (kirilklein) [reply README.md]: ok

File: README.md (hunk 3 above)
Comment (kirilklein) [reply README.md:63]: agree with Sebastian and Maria here:D

**Review by Sllambias — APPROVED** [review]
looks cool to me! :)
